sdk/core/core/src/az_http_policy.c

```diff
@@ -174,3 +174,12 @@ AZ_NODISCARD az_result az_http_pipeline_policy_transport(
 
   return send_request(p_request, p_response);
 }
+
+AZ_NODISCARD az_http_transport_options
+az_http_transport_options_default(az_http_client_send_request_fn send_request) {
```

Comment:
it is just a kind of weird that it gives the `default` options but it only sets the value it gets as parameter XD.
Looks more like an `init` for its implementation.
But I got it.  It is because of all other options we have with this `default` approach...